spec/teacher_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rspec'
+require './teacher'
+
+describe Teacher do
+  before(:each) do
+    @teacher = Teacher.new(20, 'Michael', nil, 'Software Engineering')
+  end
+
+  it 'should create a new teacher' do
+    teacher = Teacher.new(20, 'Michael', nil, 'Software Engineering')
```

Comment:
- [ ] The same happens here, if you look closely, you're repeating the creation of the teacher, please consider removing the duplication! 🤓 🔧